node/hubController.js

```diff
@@ -326,7 +327,7 @@ class HubController {
     }
 
     _createTranslator(translatorName, deviceInfo) {
-        this.ConsoleLogger.info(`_createTranslator ${translatorName}`);
+        this.logger.info(`_createTranslator ${translatorName}`);
         return this.OpenT2T.createTranslatorAsync(translatorName, deviceInfo).then( translator => {
```

Comment:
also I believe you'll have to instantiate a new logger per call from the primary entry methods and pass in here and to createTranslatorAsync. You should not pass in this.logger to createTranslatorAsync

---
In reply to: [109985579](https://github.com/openT2T/cloud/pull/32#discussion_r109985579) [](ancestors = 109985579)